src/candidateGenerator.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * Generate a new candidate
+ * @param {{lowerBound: Array<number>, upperBound: Array<number>} } neighbour - Upper and lower bound to each variable
+ * @param {number} range - Range where each value will cahnge
+ * @param {Array<number>} current - Current variables values from which a new candidate will be generated
+ * @param {number} T - value that is used to decide if new candidate is acceptable or not  
+ * @param {number} quienching - value that determine the decay rate of temperature. At high temperature almost every candidate is accepted, at low values, only good ones.  
+ */
+
+export default function candidateGenerator(neighbour,range, current, T, quienching){
```

Comment:
don't use a capital letter to start the name of a normal variable, it is reserve for classes

Reply:
Solved